syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -664,6 +664,26 @@ def get(self, *args, inplace: bool = False, user=None, reason: str = "", **kwarg
         else:
             return tensor
 
+    def get_copy(self, *args, user=None, reason: str = "", **kwargs):
```

Comment:
Q: This looks very similar to the ```get``` method...Do you have any idea how should we simplify it?
@AlanAboudib @karlhigley